py/mattak/backends/uproot/dataset.py

```diff
@@ -142,7 +142,7 @@ def __init__(self, station : int, run : int, data_path : str, verbose : bool = F
                 self._wfs = self.wf_tree[self.wf_branch]
 
                 self.hd_file = uproot.open("%s/headers.root" % (self.rundir))
-                self.hd_tree,self.hd_branch = read_tree(self.hd_file, header_tree_names);
```

Comment:
it's a good thing python ignores my semicolons!

Reply:
yes :) I was surprised by this feature!